tests/test_mpi.py

```diff
@@ -1005,6 +1011,75 @@ def test_avoid_haloupdate_if_distr_but_sequential(self, mode):
         calls = FindNodes(Call).visit(op)
         assert len(calls) == 0
 
+    @pytest.mark.parallel(mode=1)
+    def test_issue_2448(self, mode):
+        extent = (10., )
+        shape = (2, )
```

Comment:
if you are not executing the operators, you don't need Grid.extent, so you can probably just inline this variable instead